dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1673,20 +1666,20 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 	}
 
 	if (chatCommand == "reforge" && entity->GetGMLevel() >= eGameMasterLevel::DEVELOPER && args.size() >= 2) {
-		LOT baseItem;
-		LOT reforgedItem;
+		const auto baseItem = GeneralUtils::TryParse<LOT>(args[0]);
+		const auto reforgedItem = GeneralUtils::TryParse<LOT>(args[1]);
```

Comment:
nit: no point in parsing reforgedItem is baseItem didnt parse

Reply:
Can you clarify?

Reply:
Oh, see what you mean now.